scaladoc/src/dotty/tools/scaladoc/snippets/SnippetCompiler.scala

```diff
@@ -31,7 +31,7 @@ class SnippetCompiler(
       rootCtx.setSetting(rootCtx.settings.experimental, true)
       rootCtx.setSetting(rootCtx.settings.YretainTrees, true)
       rootCtx.setSetting(rootCtx.settings.YcookComments, true)
-      rootCtx.setSetting(rootCtx.settings.YreadComments, true)
+      rootCtx.setSetting(rootCtx.settings.YreadDocsEagerly, true)
```

Comment:
What is a usage scenario where I might want readDocsEargerly?

Reply:
None. I just kept it for backward compat and for someone else to be able to benchmark the difference in performance.